pages/oauth/authorize.js

```diff
@@ -59,7 +57,12 @@ const OAuthAuthorizePage = () => {
         {isLoading ? (
           <Loading />
         ) : !LoggedInUser ? (
-          <SignInOrJoinFree />
+          <SignInOrJoinFree
+            isOauth
+            oAuthAppName={data?.application?.name}
+            oAuthAppImage={data?.application?.account?.imageUrl}
+            clientId={query['client_id']}
```

Comment:
Why do we pass this `clientId`?

Reply:
Ops, I was trying a different approach and forgot to clean this up. Sorry. 🤯